framework/src/actions/AddFunctionAction.C

```diff
@@ -29,5 +29,9 @@ AddFunctionAction::AddFunctionAction(InputParameters params) :
 void
 AddFunctionAction::act()
 {
+  FunctionParserBase<Real> fp;
+  std::string vars = "x,y,z,t,NaN,pi,e";
```

Comment:
Nope, the same string appears in `FEProblemBase::getFunction`. No big deal, but this was changed before (when NaN was added by Roy).